arangod/Aql/AqlFunctionFeature.cpp

```diff
@@ -192,6 +192,7 @@ void AqlFunctionFeature::addStringFunctions() {
   add({"CONCAT", ".|+", flags, &Functions::Concat});
   add({"CONCAT_SEPARATOR", ".,.|+", flags, &Functions::ConcatSeparator});
   add({"CHAR_LENGTH", ".", flags, &Functions::CharLength});
+  add({"NORMALIZE", ".", flags, &Functions::Normalize});
```

Comment:
I think the term `NORMALIZE` is too broad, because normalization could be applied for various things, not just Unicode character sequences.
I am in favor of renaming it to `UTF8_NORMALIZE` or `NORMALIZE_UTF8`, whichever makes more sense.